tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py

```diff
@@ -15,66 +15,131 @@
           preparsing_validators=[
               validators.hasLength(71),
           ],
-          postparsing_validators=[],
+          postparsing_validators=[
+              validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.notMatches(1),
+                  result_field='SSN', result_function=validators.notOneOf(("000000000", "111111111", "222222222",
```

Comment:
I may have said this earlier, can we just make a reusable SSN validator to prevent this repetition?

Reply:
If we use the DRY idiom wrt the validators then we should be making specific validators for race/ethnicity, earned income fields, and basically every field in the T6 record to avoid duplicating the same validator. Im good with doing that, I just want to keep it consistent. Let me know how you'd like me to proceed.

Reply:
The repetition is moreso the silly-looking "00000" stuff which takes up a few lines but is repeated frequently across the RowSchemas. I'd like the SSN to have a standalone and cleaner-looking validator here. 

If we feel that there are other areas to optimize which improve readability and code smell, I'm not opposed to updating those if it's not a large lift. So, soft yes if you feel we are repeating the race/ethnicity validation.